src/test/java/io/reactivex/internal/operators/flowable/FlowableDistinctUntilChangedTest.java

```diff
@@ -141,6 +141,84 @@ public void testDistinctUntilChangedOfSourceWithExceptionsFromKeySelector() {
         inOrder.verify(w, never()).onComplete();
     }
 
+    @Test
```

Comment:
There is no reason to test this unwanted behavior.

Reply:
That's what I thought initially. Will remove all tests.

Reply:
Done.